### PR TITLE
Log DCNs

### PR DESCRIPTION
**Pull request by rkondratowicz**
Right after metadata gets successfully parsed.
https://tools.hmcts.net/jira/browse/BPS-1207

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java [hunk 1]
```diff
@@ -241,6 +242,12 @@ private void processZipFileContent(
 
             InputEnvelope envelope = envelopeProcessor.parseEnvelope(result.getMetadata(), zipFilename);
 
+            log.info(
+                "Parsed envelope. Payment DCNs: {}. Document DCNs: {}",
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java:246]: worth including zip file name and jurisdiction here ?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java]: oops. fixed.

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java [hunk 2]
```diff
@@ -241,6 +242,14 @@ private void processZipFileContent(
 
             InputEnvelope envelope = envelopeProcessor.parseEnvelope(result.getMetadata(), zipFilename);
 
+            log.info(
+                "Parsed envelope. File name: {}. Container: {}. Payment DCNs: {}. Document DCNs: {}",
+                zipFilename,
+                containerName,
+                envelope.payments.stream().map(payment -> payment.documentControlNumber).collect(joining(",")),
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java:249]: Can payments be null?

Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java:249]: Just checked, null will be converted to an empty list.

**Pull request merged**